server/plugin.go

```diff
@@ -490,3 +540,23 @@ func addFields(fields []*model.SlackAttachmentField, title, msg string, short bo
 		Short: model.SlackCompatibleBool(short),
 	})
 }
+
+func messageToJSON(message string) (bool, []uint8) {
```

Comment:
If we're returning the result of `json.Marshal`, I would expect the return type to be `[]byte`

```suggestion
func messageToJSON(message string) (bool, []byte) {
```